.github/workflows/documentation.yml

```
@@ -0,0 +1,45 @@
+name: Documentation
+
+on:
+  push:
+    branches: [ development ]
+  pull_request:
+    branches: [ development ]
+  
+jobs:
+  make-documentation:
+    runs-on: ubuntu-latest
```

I would fix the version of ubuntu just to be on the safe side
```suggestion
    runs-on: ubuntu-20.04
```